README.md

```diff
@@ -1,89 +1,32 @@
-# Important Links
-- **Deployed App:** https://csce606arcade-12ac8dd4dc24.herokuapp.com/users
-- **Code Climate Report:** https://codeclimate.com/github/tamu-edu-students/csce600-arcade
-- **Project Management Page:** https://tamu-team-pyr0027e.atlassian.net/jira/software/projects/SCRUM/boards/1
+# ARCADE
```

Comment:
thank you for doing this! I completely missed that part of the assignment and read it as put it in your read me